vulture/core.py

```diff
@@ -46,6 +47,28 @@
 if sys.version_info < (3, 4):
     IGNORED_VARIABLE_NAMES |= {"True", "False"}
 
+ERROR_CODES = {
+    "attribute": "V001",
+    "class": "V002",
+    "function": "V003",
+    "import": "V004",
+    "property": "V005",
+    "unreachable_code": "V006",
+    "variable": "V007",
+}
```

Comment:
I think we should follow pycodestyle and start with error code V101 instead of V001. Since unreachable_code is somewhat different from unused code, I'd give it number V201 (and unused variables V107). (I see now that you changed the order.)

Reply:
If we start with V101, and don't assign unreachable_code in that series, unused_variables would be V106, no?

Reply:
That's right :-)